index.php

```diff
@@ -95,6 +95,11 @@
 		] ),
 	]
 ] );
+
+$banner = banner_html();
+if ( $banner ) {
+	echo "<p class='banner'>$banner</p>";
```

Comment:
Should we use a MessageWidget with type=warning (or info)?

Reply:
Also, should the banner go above the form, given that it should be read before creating a wiki?

Reply:
I don't want to put it at the top because I think no one will read it then :D I was considering putting it in the middle of the form though (above the button), but I like it here.

I greatly dislike how MessageWidget looks.

Reply:
We should call this "[new/create]Wiki[Banner/Warning]", as "banner" suggests is appears everywhere.

If we add other banners later (e.g. on new.php, or Main_Page) that's one less thing to change in your config.